assets/material-design.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener("DOMContentLoaded", calculateDeliveryDays());
+
+function calculateDeliveryDays() {
+    let deliveryTime = document.querySelectorAll("[data-delivery-time]")[0].dataset.deliveryTime;
+
+    // getDay: 0 - sunday, 6 - saturday, 1-5 - normal
+    const workDays = [1, 2, 3, 4, 5];
+    const milisecondsInDay = (3600 * 1000 * 24);
+
+    let suffix = "days";
+    let index = 0;
+    let calculatedDeliveryTime = 0;
+
+    while (index <= deliveryTime) {
+        var date = new Date(Date.now() + milisecondsInDay * calculatedDeliveryTime);
+        var day = date.getDay();
```

Comment:
Variable declarations - you have used both **let** and **var**. Good practice is to use **let** consistently.

Reply:
Thanks for pointing me this good practice